### PR TITLE
Add helper class to have methods used multiple times

### PR DESCRIPTION
**Pull request by saadsahibjan**
PR for #23

### CODE REVIEW

File: routes/user-api.js [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 //init the required modules
 var express = require('express');
-var log = require('debug-logger')('user-api.js');
-var validator = require('validator');
-var crypto = require('crypto');
+var log = require('debug-logger')('user-api.js');;
```
Comment (murzeus) [inline routes/user-api.js]: having two semicolons at the end?

File: utils/helper.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+
+    encryptPassword: function(password) {
+        var cipher = crypto.createCipher('aes192', CIPHER_SALT);
+        var encryptedPass = cipher.update(password, 'utf-8', 'hex');
+        encryptedPass += cipher.final('hex');
+        return encryptedPass;
+    },
+
+    checkEmail: function(email, res) {
```
Comment (murzeus) [inline utils/helper.js]: let's make this as a generic function to check whether the given property is undefined or not.

Comment (saadsahibjan) [inline utils/helper.js]: That's a good idea. Il rename the method.

Comment (murzeus) [inline utils/helper.js]: yeah it should be some thing like `hasPropertyof`where you can pass the object and a specific property you want to check.

Comment (saadsahibjan) [inline utils/helper.js]: Fixed in commit https://github.com/deleepa/simple-api-server/pull/25/commits/5858bc0da061efd3a1412ae08aa4e8172a3b2e0f of the same PR

File: routes/user-api.js (hunk 1 above)
Comment (saadsahibjan) [inline routes/user-api.js]: Fixed in commit https://github.com/deleepa/simple-api-server/pull/25/commits/5858bc0da061efd3a1412ae08aa4e8172a3b2e0f of the same PR

**Pull request merged**